src/components/EditCoursePage/CollapsibleCourseRunFields.scss

```diff
@@ -0,0 +1,3 @@
+.collapsible-body {
+  overflow: visible !important;
+}
```

Comment:
I was looking for whether we could just do this in a bootrstrap class, and didn't see anything in my quick scan. But I did came across:

https://getbootstrap.com/docs/4.0/components/dropdowns/#options

Might be useful when we get to the "dropping up" part of this work.